bokeh/models/tools.py

```diff
@@ -733,6 +733,13 @@ class ZoomInTool(ActionTool):
     Percentage to zoom for each click of the zoom-in tool.
     """)
 
+    maintain_focus = Bool(default=True, help="""
+    Whether or not zooming tool maintains its focus position. Setting to False
+    results in a more "gliding" behavior, allowing one to zoom out more
```

Comment:
Whatever this property is named, this doctring will need tweaking. The zoom-in and zoom-out buttons are never really "gliding" in any context (I think that clause should be removed in the wheel zoom tool too to be honest, it is not a very good description)

Reply:
I have updated the docstrings, let me know if they are better now. I also took the liberty of adjusting the WheelZoom docstring as per your comment.

Reply:
With regards to the name: I like maintain_focus better than your other suggestion (independent_bounds or bounds_policy), but will use whatever is deemed best.